sdk/src/client/node_api/core/routes.rs

```diff
@@ -349,6 +348,56 @@ impl ClientInner {
             .await
     }
 
+    // Commitments routes.
+
+    /// Gets the slot commitment by the given slot commitment id.
+    /// GET /api/core/v3/commitments/{commitmentId}
+    pub async fn get_slot_commitment_by_id(&self, slot_commitment_id: &SlotCommitmentId) -> Result<SlotCommitment> {
+        let path = &format!("api/core/v3/commitments/{slot_commitment_id}");
+
+        self.node_manager
```

Comment:
can we just reuse the `get_slot_commitment_by_id_raw`?

Reply:
We could but I'd rather not, not sure we have the guarantee the raw version will exist on all nodes, etc...

Reply:
aa ok makes sense